comb_spec_searcher/proof_tree.py

```diff
@@ -712,6 +712,53 @@ def _recursion_fixer(self, css, root=None, in_labels=None):
         for child in root.children:
             self._recursion_fixer(css, child, in_labels)
 
+    def expand_tree(self, pack, **kwargs):
+        """
+        Return a ProofTree that comes from expanding the strategy verified
+        combinatorial classes using the StrategyPack 'pack'.
+
+        The function relies on CombinatorialSpecificationSearcher and its
+        method 'auto_search'. It first created a universe with rules from
+        'self', and then adds strategy_verified objects to the front of the
+        queue. All classes are assumed to be expandable. The '**kwargs' are
```

Comment:
The kwargs are also passed to the Searcher init

Reply:
@christianbean I think the kwargs are not needed anywhere here. Maybe we just remove them

Reply:
You might want to add status updates, or max time for searching.

Reply:
Are you proposing that we drop the kwargs for the searcher and only keep them for the autosearch?

Reply:
I propose leaving it as is, and give the kwargs to both.